pkg/plugins/golang/v4/scaffolds/internal/templates/readme.go

```diff
@@ -111,6 +116,25 @@ You can apply the samples (examples) from the config/sample:
 
 %s
 
+## Distributing your project
+
+Following the steps for you build the installer and to distribute this project for users.
+
+### Build the installer
+To build the installer for the image built and published in the registry:
+
+make build-installer IMG=<some-registry>/{{ .ProjectName }}:tag
+
+NOTE: The above makefile target will generate the install.yaml in the directory dist contains all resources
+build with Kustomize to install this project.
+
+### To install via the installer
+
+You must just run kubectl apply -f dist/install.yaml.
+Therefore, users can install this project if they have access to the repository by running:
+
```

Comment:
Nits:


To install this project, simply run the following command:

```sh
kubectl apply -f dist/install.yaml
```

Additionally, users with access to the repository can install this project by executing:

```sh
kubectl apply -f https://raw.githubusercontent.com/<org>/project/<tag or branch>/dist/install.yaml
```

Reply:
We are already adding above the make targets about how to install.
The goal here is just to show how the project can be distributed without the user needing to git clone the solution. 
I will see how we can improve the info to make it clear.

Reply:
Sounds good!